<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ahdog</class>
 <widget class="QMainWindow" name="ahdog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1662</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1581</width>
      <height>841</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>概览</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>1431</width>
        <height>631</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>细节</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>551</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Acode</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>当前比价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Aprice</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hprice</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>70</y>
        <width>861</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>均值</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5%分位</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>10%分位</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>75%分位</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>90%分位</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>max</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>min</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>std</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1662</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
